tox.ini

```diff
@@ -6,10 +6,14 @@ basepython = python3
 setenv =
     USING_TOX=1
 deps =
-    -r{toxinidir}/requirements.txt
-    -r{toxinidir}/tests/requirements.txt
+    -r tests/requirements.txt
 commands =
-    pytest --verbose --cov=shub --cov-report=term-missing --cov-report=html --cov-report=xml {posargs:shub tests}
+    pytest --cov=shub --cov-report=term-missing --cov-report=html --cov-report=xml {posargs:shub tests}
+
+[testenv:min]
+deps =
+    {[testenv]deps}
+    -r requirements.txt
```

Comment:
A separate Tox environment to test those “minimum” versions of dependencies, while keeping the rest of the Tox environments testing the latest versions of dependencies.